components/Vote/VoteCounter.js

```diff
@@ -0,0 +1,275 @@
+                  <div
+                    key={i}
+                    {...merge(styles.goal, i === 0 && styles.currentGoal)}
+                    {...colorScheme.set('borderBottomColor', 'secondaryBg')}
+                    {...colorScheme.set('borderRightColor', 'default')}
+                    style={{
+                      width: widthForGoal(endGoal.number, currentGoal.number)
+                    }}
+                    onTouchStart={e => {
+                      e.preventDefault()
+                      setHover(currentGoal)
+                    }}
+                    onTouchEnd={() => setHover(undefined)}
+                    onMouseOver={() => setHover(currentGoal)}
+                    onMouseOut={() => setHover(undefined)}
+                  >
+                    {hover && currentGoal.number === hover.number && (
+                      <div {...styles.noInteraction}>
+                        <div
+                          {...styles.goalBar}
+                          {...colorScheme.set('backgroundColor', 'secondary')}
+                          style={{
+                            width: widthForGoal(currentGoal.number, submitted)
+                          }}
+                        />
+                        <div
+                          {...styles.goalNumber}
+                          {...colorScheme.set('color', 'secondary')}
+                          {...colorScheme.set('borderRightColor', 'secondary')}
+                        >
+                          {format(currentGoal.number)}
+                        </div>
+                        {!!hover.description && (
+                          <div
+                            {...styles.arrow}
+                            {...colorScheme.set(
+                              'borderBottomColor',
+                              'secondary'
+                            )}
+                          />
+                        )}
+                      </div>
+                    )}
+                  </div>
+                ))}
+              {!!hover && !!hover.description && (
+                <div
+                  {...styles.box}
+                  {...colorScheme.set('backgroundColor', 'secondary')}
+                >
+                  <span
+                    style={{ color: '#fff' }}
+                    dangerouslySetInnerHTML={{
+                      __html: hover.description.replace(
+                        '{count}',
+                        format(hover.number)
+                      )
+                    }}
+                  />
+                </div>
+              )}
+            </div>
+            {showCountdown && (
+              <VoteCountdown
+                endDate={voting.endDate}
+                caption={countdownCaption}
+                over={countdownOver}
+              />
+            )}
+          </div>
+        )
+      }}
+    />
+  )
+}
+
+export default compose(graphql(query))(GoalBar)
```

Comment:
Mini feature request:

Allow setting a [`pollInterval`](https://www.apollographql.com/docs/react/v2.6/api/react-hoc/#optionspollinterval) on the query:

```suggestion
export default compose(graphql(query, {
  options: ({ pollInterval }) => ({
    pollInterval: pollInterval || 0
  })
}))(GoalBar)
```